src/auditlog/diff.py

```diff
@@ -24,7 +24,11 @@ def track_field(field):
         return False
 
     # Do not track relations to LogEntry
-    if getattr(field, 'rel', None) is not None and field.rel.to == LogEntry:
+    if getattr(field, 'remote_field', None) is not None and field.remote_field.model == LogEntry:
+        return False
+
+    # 1.8 check
+    elif VERSION == (1, 8) and getattr(field, 'rel', None) is not None and field.rel.to == LogEntry:
```

Comment:
I don't really like the explicit version check, wouldn't it be sufficient to just do the `getattr()` call?